gluster-file-system/azuredeploy.json

```diff
@@ -0,0 +1,318 @@
+﻿{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "string",
+            "defaultValue": "West US",
+            "metadata": {
+                "description": "Location where the nodes will be created"
+            }
+        },
+        "storageAccountName": {
+            "type": "string",
+            "metadata": {
+                "description": "Prefix of storage account name, each node lives in its own storage account"
+            }
+        },
+        "scaleNumber": {
+            "type": "int",
+            "defaultValue": 2,
+            "allowedValues": [ 2, 4, 6, 8 ],
+            "metadata": {
+                "description": "Number of nodes in the gluster file system"
+            }
+        },
+        "virtualNetworkName": {
+            "type": "string",
+            "metadata": {
+                "description": "virtual network the nodes will be created in"
+            }
+        },
+        "adminUsername": {
+            "type": "string",
+            "metadata": {
+                "description": "ssh user name"
+            }
+        },
+        "adminPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "ssh password"
+            }
+        },
+        "imagePublisher": {
+            "type": "string",
+            "defaultValue": "OpenLogic",
+            "allowedValues": [ 
+                "OpenLogic"
+            ],
+            "metadata": {
+                "description": "the publisher name of the OS image"
+            }
+        },
+        "imageOffer": {
+            "type": "string",
+            "defaultValue": "CentOS",
+            "allowedValues": [ 
+                "CentOS"
+            ],
+            "metadata": {
+                "description": "the offer name (OS family) of the OS image"
+            }
+        },
+        "imageSku": {
+            "type": "string",
+            "defaultValue": "6.5",
+            "allowedValues": [ 
+                "6.5"
+            ],
+            "metadata": {
+                "description": "the SKU name of the image offering typically major OS version"
+            }
+        },
+        "vmSize": {
+            "type": "string",
+            "defaultValue": "Standard_A1",
+            "allowedValues": [
+                "Standard_A1",
+                "Standard_A2",
+                "Standard_A3",
+                "Standard_A4"
+            ],
+            "metadata": {
+                "description": "VM size for the nodes"
+            }
+        },
+        "vmNamePrefix": {
```

Comment:
If user already has VMs with the same name in this resource group, the deployment will fail, right?